### PR TITLE
Fix #46157

### LINKED ISSUE #46157: VB Code inside Roslyn can't use latest VB features
**Version Used**: 
VS Version 16.7.0 Preview 4.0
Roslyn Master 7/20/2020
**Steps to Reproduce**:
Open any VB File on insert _ ' Test on any line before a space, example below
image

**Expected Behavior**:
No error
**Actual Behavior**:
Error BC37306
This does not happen on other VB projects

### PR DESCRIPTION
**Pull request by paul1956**
Allow Roslyn Code to use features of VB16

### CODE REVIEW

File: src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb [hunk 1]
```diff
@@ -76,7 +76,8 @@ End Class
                            </Project>
                        </Workspace>
 
-            AssertCommitsStatement(test, expectCommit:=False)
+            AssertCommitsStatement(test, _ ' Test that Roslyn supports VB16 Continuation Comments After Line Continuation internally
+                                   expectCommit:=False)
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb:80]: can you undo this, it seems unnecessary.

Comment (CyrusNajmabadi) [reply src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb:80]: (and i don't think is a style we would allow in teh codebase).

Comment (paul1956) [reply src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb:80]: It's not, It is just a test that proves VB16 works. The change to VB16 is needed for #5056. Do you Just not want any tests?

Comment (CyrusNajmabadi) [reply src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb:80]: > Do you just not want any tests?

This isn't really a test.  You're editing actual code here.  I don't like thsi style and would strongly push for it to not be allowed in the IDE codebase.  Thanks :)

Comment (paul1956) [reply src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb]: Deleted will allow anything that depends on VB16 to be the place that complains if VB LangVersion ever set <16

**Pull request merged**